typespec-extension/src/code-model-builder.ts

```diff
@@ -1409,7 +1409,8 @@ export class CodeModelBuilder {
             // parameter
             if (
               existParameter.implementation === ImplementationLocation.Method &&
-              (existParameter.origin?.startsWith("modelerfour:synthesized/") ?? true)
+              (existParameter.origin?.startsWith("modelerfour:synthesized/") ?? true) &&
+              !(existParameter.schema instanceof ConstantSchema)
             ) {
```

Comment:
Bug was here.

Now we just don't add the param to convenience API if it is constant.